portlet-security-util/src/main/java/org/jasig/web/filter/SimpleCorsFilter.java

```diff
@@ -0,0 +1,133 @@
+package org.jasig.web.filter;
+
+import java.io.IOException;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang.StringUtils;
+
+/**
+ * Default behavior for this filter is to allow origins from * (ie anywhere). It is not reccomended to use this default 
+ * behavior except for development testing. When configuring the filter, the filter configuration parameters will 
+ * overwrite any value set in a spring or other configuration file doing dependency injection and initialization.  
+ * 
+ * Config within web.xml:
+ * 
+ * <filter>
+ *   <filter-name>Simple CORS Filter</filter-name>
+ *   <filter-class>org.jasig.web.filter.SimpleCorsFilter</filter-class>
+ *   <init-param>
+ *     <!-- Comma separated domain names -->
+ *     <param-name>allowOrigin</param-name>
+ *     <param-value>someDomain.org, another.com, jasig.org, apereo.org</param-value>
+ *     
+ *     <param-name>maxAge</param-name>
+ *     <param-value>3600</param-value>
+ *     
+ *     <!-- Comma separated methods allowed -->
+ *     <param-name>allowMethod</param-name>
+ *     <param-value>POST, GET</param-value>
+ *     
+ *     <!--Comma separated list of allowed header values -->
+ *     <param-name>allowHeaders</param-name>
+ *     <param-value>Origin, X-Requested-With, Content-Type, Accept</param-value>
+ *   </init-param>    
+ * </filter>
```

Comment:
As is this is a straight-up servlet filter.  This means you have to list all the domains for all environments (dev, test, prod, etc.) in the portlet's web.xml or have a substitution process.  Instead I suggest implementing a Spring HandlerInterceptor.  This allows you to inject the values from a properties file using Spring.  See http://docs.spring.io/spring/docs/4.1.8.RELEASE/spring-framework-reference/htmlsingle/#mvc-handlermapping-interceptor.